sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/datastore/DatastoreV1.java

```diff
@@ -697,7 +910,7 @@ public DeleteKey deleteKey() {
      * Note that {@code projectId} is only {@code @Nullable} as a matter of build order, but if
```

Comment:
This comment repeats a lot of times. What can be done to reduce the duplication?